src/common/InputRowRadioButtons.js

```diff
@@ -164,7 +170,7 @@ export default function InputRowRadioButtons<TItemKey: string>(props: Props<TIte
           <ZulipTextIntl text={selectedItem.title} style={styles.valueTitle} />
         </View>
         <View style={styles.iconRightWrapper}>
-          <IconRight size={kRightArrowIconSize} color={BRAND_COLOR} />
+          <IconRight size={kRightArrowIconSize} color={themeData.color} />
```

Comment:
What's the effect of this first commit -- is there an existing screen where it changes something?

Reply:
Yep, the edit-stream screen. Here's a before-and-after:

<img width=300 src="https://user-images.githubusercontent.com/22248748/187572009-0bb23c6a-0317-4753-ab8f-a35fa05f9aa4.png" /> <img width=300 src="https://user-images.githubusercontent.com/22248748/187571988-8a9d9163-60cd-413d-bd71-b0b68bc41931.png" />

Reply:
Great, thanks. Seems good.